basis_universal.lua

```diff
@@ -0,0 +1,631 @@
+project "basis_universal"
+
+local prj = project()
+local prjDir = prj.basedir
+
+-- -------------------------------------------------------------
+-- project
+-- -------------------------------------------------------------
+
+-- common project settings
+
+dofile (_BUILD_DIR .. "/3rdparty_static_project.lua")
+
+-- project specific settings
+
+uuid "E2587158-A6F8-4BF3-9A7D-9CB1C2F0A5FE"
+
+flags {
+  "NoPCH"
+}
+if not (_PLATFORM_WINDOWS or _PLATFORM_WINUWP) then
+  buildoptions {
+    "-Wall",
+    "-Wno-reorder",
+    "-Wno-unknown-pragmas",
+    "-Wno-undefined-inline",
+    "-Wno-unused-variable",
+  }
+end
+
+files {
+  "encoder/**.h",
+  "encoder/**.cpp",
+  "transcoder/**.h",
+  "transcoder/**.cpp",
+  "*.h",
```

Comment:
I don't see any .h or .cpp files at the root. Are these needed?

Reply:
https://github.com/Esri/basis_universal/tree/bori8447/basis_universal_for_runtimecore

I see one .c, 18 .h and 17 .cpp files.

Reply:
Ah, my mistake. I was looking at the master branch which they've put into folders. My bad.